### PR TITLE
Add android compilation to travis

### PR DESCRIPTION
**Pull request by MortimerGoro**
Add android compilation to travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,3 +26,5 @@ script:
   # - cargo test --verbose
   # - cargo test --verbose --features test_egl_in_linux
   - if [ "$TRAVIS_OS_NAME" != "linux" ]; then cargo test --verbose; fi
+  - cargo build --verbose --target=arm-linux-androideabi
```
Comment (emilio) [inline .travis.yml]: You probably need to `rustup install arm-linux-androideabi` before.

**Review by emilio — APPROVED** [review]
This is currently red, but thanks for this!

**Review by emilio — CHANGES_REQUESTED** [review]
Err, wrong checkbox.

**Comment by MortimerGoro** [comment]
This was already fixed fixed.

**Pull request merged**